src/api/app/models/bs_request_action.rb

```diff
@@ -921,7 +925,7 @@ def check_permissions_for_sources!
   end
 
   # find default reviewers of a project/package via role
-  def find_reviewers(obj)
+  def find_reviewers(obj, disable_project = false)
```

Comment:
```suggestion
  def find_reviewers(obj, disable_project: false)
```

Sorry. I meant this. RuboCop complained.

Reply:
done